framework/db/pgsql/JsonExpressionBuilder.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace yii\db\pgsql;
+
+use yii\db\ExpressionBuilderInterface;
+use yii\db\ExpressionBuilderTrait;
+use yii\db\ExpressionInterface;
+use yii\db\JsonExpression;
+use yii\db\Query;
+use yii\helpers\Json;
+
+/**
+ * Class JsonExpressionBuilder builds [[JsonExpression]] for PostgreSQL DBMS.
+ *
+ * @author Dmytro Naumenko <d.naumenko.a@gmail.com>
+ * @since 2.0.14
+ */
+class JsonExpressionBuilder implements ExpressionBuilderInterface
+{
+    use ExpressionBuilderTrait;
+
+    const PARAM_PREFIX = ':qp';
+
+    /**
+     * {@inheritdoc}
+     * @param JsonExpression|ExpressionInterface $expression the expression to be built
+     */
+    public function build(ExpressionInterface $expression, &$params = [])
+    {
+        $value = $expression->getValue();
+
+        if ($value instanceof Query) {
+            list ($sql, $params) = $this->queryBuilder->build($value, $params);
+            return "($sql)" . $this->getTypecast($expression);
```

Comment:
It is good that params aren't used?

Reply:
This variable is passed by reference, so it is OK